src/signald_client/main.py

```diff
@@ -15,34 +14,60 @@
 class Signal(_Signal):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+        self.logger = logging.getLogger("SignalListener")
         self._chat_handlers = []
         self._payment_handlers = []
+        self._run = True
+
+    def isolated_handler(self, func):
```

Comment:
Can we add docstrings here for both isolated_handler and register_handler?

Reply:
Done :)